tests/unit_tests/cpu/material/stopping_power.cpp

```diff
@@ -0,0 +1,147 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "detray/materials/interaction.hpp"
+#include "detray/materials/material.hpp"
+#include "detray/materials/predefined_materials.hpp"
+
+// Detray test include(s)
+#include "detray/test/utils/types.hpp"
+
+// GTest include(s).
+#include <gtest/gtest.h>
+
+using namespace detray;
+
+// Test class for the stopping power
+// Input tuple: < material, particle type, kinetic energy, expected output >
+class StoppingPowerValidation
+    : public ::testing::TestWithParam<
+          std::tuple<material<scalar>, pdg_particle<scalar>, scalar, scalar>> {
+};
+
+TEST_P(StoppingPowerValidation, stopping_power) {
+
+    // Interaction object
+    interaction<scalar> I;
```

Comment:
Please use the ```test::scalar``` type here, as I am trying to get rid of the global scalar typedef (it clashes if we want to use multiple algebra plugins in the same src)